src/http-api/routes/pubsub.js

```diff
@@ -1,38 +1,23 @@
 'use strict'
 
-const Joi = require('joi')
 const resources = require('./../resources')
 
 module.exports = (server) => {
   const api = server.select('API')
 
   api.route({
     method: '*',
-    path: '/api/v0/pubsub/sub/{topic}',
+    path: '/api/v0/pubsub/sub',
     config: {
-      handler: resources.pubsub.subscribe.handler,
-      validate: {
-        params: {
-          topic: Joi.string().required()
-        },
-        query: {
-          discover: Joi.boolean()
-        }
-      }
+      handler: resources.pubsub.subscribe.handler
```

Comment:
@dignifiedquire - I like the resulting cleanliness / uniformness of this change. Was the primary driver the fact that you wanted cleaner error messages?